### DISCUSSION TITLE
Weekly sprint 23/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#223] [General]
Minuta de la reunión:

- Quiero tener el video del final mucho antes de la fecha límite, el 14 cierra el sprint y el domingo anterior es 13 pero no quiero esperar hasta el último momento por cualquier problema de conexión o de edición que haya así que la idea es grabarlo el 6 de Noviembre durante la reunión semanal. Los que no puedan estar el 6/11 a la mañana pueden conectarse a un segundo meet a la noche el mismo 6/11 para grabar su parte. Si no pueden estar el 6/11 para nada tendrán que enviar una grabación ustedes mismos dentro de los parámetros pedidos (resolución de 1920x1080, grabación de voz y pantalla, formato mp4, etc). Tenemos una discusión abierta con las secciones del sitio para que elijan más o menos de qué hablar y no se pisen con alguien más.
- En la tarea de videos pueden dejar puesto el script o resumen que van a leer mientras hacen la presentación, pueden probar haciendo demostraciones en su propia máquina y mencionar cuánto les lleva, de esa manera tenemos una idea de cuánto tiempo tiene el resto para hablar para que el video no pase de los 10 minutos.
- Hay tareas abiertas con los queries que hay que armar para las consultas del sitio, lo ideal es que elijan uno o más queries para realizar. Una vez que arman el query y lo ponen en la tarea puden agregarlo a model/queries.sql que va a tener la lista de todos los queries que vamos a usar.
- También hay tareas para completar los "DTOs" en Python, estas clases representan registros de la tabla (representado por la versión en plural, Animal es el registro y Animales es la tabla por poner un ejemplo), intenten tomar uno y hacerlo como mínimo. Si además pueden hacer la clase que representa la tabla dentro del mismo archivo mejor todavía.
- También hay tareas para los controladores que obtendrían los datos de la página web, si tienen experiencia en desarrollo web pueden intentar implementarlos. Los controladores de registración, login y recuperación de claves ya están funcionando, el resto por el momento no es necesario que funcionen.
- La parte de controlador tiene pruebas unitarias, sin embargo no es necesario que las escriban, en todo caso las escribiré yo luego ya que no son requeridas.
- Todavía no tenemos información sobre si hay que grabar los datos en base de datos, en caso de que así sea lo veremos durante la semana o el domingo que viene.
- Tienen que agregar javascript al sitio en algún lado, tenemos una discusión con ideas sobre dónde y qué tipo de JS agregar, vayan eligiendo qué hacer.
- Se actualizó el README con información sobre cómo correr el servidor por si quieren probar la registración y el login.
- Todavía no encaramos lo del sitio de Almoneda, si alguien sabe Wordpress puede empezar con eso, si nadie puede lo voy a hacer a último momento.

Tengan en cuenta que el backlog es del equipo así que en todo lo que puedan ayudar bienvenido sea, no es necesario que les asigne tareas para hacer. Cada uno puede ver la cantidad de commits que hizo en [Insights](https://github.com/rpgrca/proyecto-ispc-fullstack/graphs/contributors) y puede decidir si lo que hicieron en comparación a la envergadura del sistema justifica la promoción o siquiera la aprobación del trabajo práctico.

**Comment by NataliaAlvarezIspc** [comment]
Queremos!
Tenemos!
